package.json

```diff
@@ -34,6 +34,7 @@
     "publish:beta": "lerna publish --exact --dist-tag=beta --preid=beta --conventional-commits --conventional-prerelease --message 'chore(release): Publish [ci skip]' --no-verify-access --yes",
     "publish:release": "lerna publish --conventional-commits --exact --yes --message 'chore(release): Publish [ci skip]' --no-verify-access",
     "postpublish:release": "git fetch . release:master && git push origin master",
+    "publish:layers": "lerna publish --exact --dist-tag=layers --preid=alpha --conventional-commits --conventional-prerelease --message 'chore(release): Publish [ci skip]' --no-verify-access --yes",
```

Comment:
I think this needs to be renamed by the branch name since the deploy process will run `npm run publish:$CIRCLE_BRANCH`.

Reply:
Yes, it has to match the branch name